src/shared/browser-compatibility-utils.js

```diff
@@ -1,10 +1,11 @@
 /**
  * Normalize a keyboard event key name.
  *
- * Several old browsers, such as IE11, use alternate key
- * names for keyboard events. If any abnormal keys are used,
- * this method returns the normalized name so our UI
- * components don't require a special case.
+ * Some old Microsoft browsers, such as IE 11 and Edge Legacy [1], use non-standard
+ * names for some keys. If any abnormal keys are used, this method returns the
+ * normalized name so our UI components don't require a special case.
+ *
+ * [1] https://caniuse.com/keyboardevent-key
```

Comment:
I haven't actually tested that Edge Legacy still needs this, I'm relying on caniuse.com's data which says it does.

Reply:
Thanks for being so thorough here!